src/init/features/dataconnect/index.ts

```diff
@@ -425,12 +425,33 @@ async function promptForCloudSQL(setup: Setup, info: RequiredInfo): Promise<Requ
   return info;
 }
 
-async function promptForService(info: RequiredInfo): Promise<RequiredInfo> {
+async function promptForSchema(setup: Setup, info: RequiredInfo): Promise<RequiredInfo> {
   if (info.serviceId === "") {
     info.serviceId = await input({
       message: "What ID would you like to use for this service?",
       default: basename(process.cwd()),
     });
+    logBullet(
+      `Check out the terms of service of Gemini in Firebase https://firebase.google.com/docs/gemini-in-firebase/set-up-gemini#required-permissions`,
+    );
+    if (
```

Comment:
```suggestion
    if ( setup.projectId && 
```
We should support init without project id, so make sure that we don't offer this flow there (since it needs a project)

Reply:
Good catch!